elements/pfe-navigation-account/demo/index.html

```diff
@@ -51,7 +51,7 @@
   <body unresolved>
     <h1><code>pfe-navigation-account</code></h1>
 
-    <pfe-navigation-account logout-link="#log-out">
+    <pfe-navigation-account lang="en" logout-link="#log-out">
```

Comment:
Love this!  Another possibility is to perform a lookup to the html tag for the language code so that Drupal can set it without having to put it directly on the component. https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang

Maybe this is a #feature idea to include support for that but it's a great source of truth that requires less mark-up.

Reply:
💯